src/api/ftl.c

```diff
@@ -305,21 +305,28 @@ int get_system_obj(struct ftl_conn *api, cJSON *system)
 	JSON_ADD_ITEM_TO_OBJECT(system, "sensors", sensors);
 
 	// Try to obtain device model
-	FILE *f_model = fopen("/sys/firmware/devicetree/base/model", "r");
-	char model[1024] = { 0 };
-	if(f_model && fgets(model, sizeof(model)-1, f_model))
+	if(file_exists("/sys/firmware/devicetree/base/model"))
```

Comment:
In `PADD` we have a more extensive way of identifying the device model than just looking at `/sys/firmware/devicetree/base/model`.

Do you think this is something to consider for FTL as well?

https://github.com/pi-hole/PADD/blob/a9681264c83529a2bd826bc86dd933d85b9b15bf/padd.sh#L214-L250

Cleaning of unwanted OEM information could still happen at the client side.

Reply:
![Screenshot from 2023-01-28 07-48-21](https://user-images.githubusercontent.com/16748619/215251561-acab7729-d92e-4a19-b5b4-a81250169077.png)

Reply:
O.E.M. is strong with you ;-)

Reply:
It's a cheap yet extremely powerful Ryzen 5 powered barebone device so one might argue that I am the OEM.